Doc/articles/features/toasts.md

```diff
@@ -0,0 +1,7 @@
+# Uno Support for Windows.UI.Notifications.Toast*
+
+### Limitations

```

Comment:
Codacy found an issue: [[heading-increment] Heading levels should increment by one level at a time](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4930649)